recipes/stdgpu/all/cmake/stdgpu-dependencies-cuda.cmake

```diff
@@ -0,0 +1,6 @@
+# Mimics the behavior of stdgpu-dependencies.cmake exported by stdgpu for the CUDA backend
```

Comment:
The project looks active, is it possible to send it to the upstream?

Reply:
Possibly. I think it already does something similar already, though. I'll have to look into it.
Either way, these CMake modules are unavoidable in this recipe due to CUDA and HIP not being available as packages, so these are needed to correctly configure them as dependencies.